book/website/reproducible-research/overview/overview-barriers.md

```diff
@@ -0,0 +1,179 @@
+
+```{figure} ../../figures/help-you-of-the-future.jpg
+---
+width: 500px
+name: help-you-of-the-future
+alt: A cartoon of a woman passing a folder of documents back to herself. Speech bubble says You're mainly keeping records for you in the future.
+---
+Although making clear documentation may feel like it is taking a lot of time in the moment, you are helping your and your collaborators remember what you have done so it is easy to reuse the work or make changes in the future.
+Illustration by The Ludic Group LLP from Kirstie Whitaker's keynote presentation at Scientific Data in 2017.
+Used under a CC-BY 4.0 license.
+DOI: [10.6084/m9.figshare.5577340.v1](https://doi.org/10.6084/m9.figshare.5577340.v1)
+```
+
+However, _The Turing Way_ community advocates that this time is more than made up for by the end of the project.
+Take as a thought experiment a reviewer asking for "just one more analysis" when the publication has been submitted to a journal.
+In many cases this request will come 6 to 12 months after the research team have worked with the raw data.
+It can be very hard to go back in time to find the one part of the pipline that the reviewer has asked you to change.
+If the work is fully reproducible, including with version controlled data and figure generating code, this analysis will be very fast to run and incorporate into the final research output.
+The analysis pipeline can be easily adapted as needed in response to co-author and reviewer requests.
+It can also be easily reused for future research projects.
+
+## Support additional users
+
+Many people worry that by making their analysis reproducible they will be required to answer lots of questions from future users of their code.
+These questions may cover software incompatibility across operating systems and the dependencies changing over time (see the [Big data and complex computational infrastructure](#big-data-and-complex-computational-infrastructure) barrier above).
+They may also include questions about how to adjust the code for a different purpose.
+
+This barrier is based in part on conflating "reproducible" with "open" research.
+The _Turing Way_ [definition of "reproducible"](../definitions) doesn't require authors to support the expansion and re-use of the data and code beyond running the exact analyses that generate the published results in the accompanying manuscript.
+
+In almost all cases, making code and data open source requires better documentation than a researcher would write for themselves.
+This can feel like an additional barrier, although - as discussed in the previous section on reproducible research [taking extra time](#takes-time) it is likely that the primary beneficiaries of well commented and tested code with detailed documentation are the research team - particularly the principle investigator of the project - themselves.
```

Comment:
```suggestion
In almost all cases, making code and data open source requires better documentation than a researcher would write for themselves.
This can feel like an additional barrier, although - as discussed in the previous section on reproducible research [taking extra time](#takes-time) it is likely that the primary beneficiaries of well commented and tested code with detailed documentation are the research team - particularly the principal investigator of the project - themselves.
```